controllers/farmer/getFarmers.js

```diff
@@ -0,0 +1,64 @@
+const { models } = require('../../models');
+const { createError, NOT_FOUND } = require('../../helpers/error.js');
+
+/**
+ * @description Get Farmers
+ *
+ * @param {object} req
+ * @param {object} res
+ * @param {object} next
+ */
+
+const getAllFarmers = async (req, res, next) => {
+  try {
+    const farmers = await models.Farmer.find({});
+    if (farmers === {}) {
+      return res.status(404).json({
+        succes: false,
+        message: 'Could not find any farmer in the record'
+      });
+    }
+    return res.status(200).json({
+      success: true,
+      message: 'Farmers records found',
+      farmers
+    });
+  } catch (err) {
+    return next(
+      createError({
+        message: 'Farmers records not found',
+        status: NOT_FOUND
+      })
+    );
+  }
+};
+
+const getFarmerById = async (req, res, next) => {
+  try {
+    const { id } = req.params.id;
+    const farmer = await models.Farmer.findById(req.params.id);
+    if (id) {
+      return res.status(200).json({
+        success: true,
+        message: 'Farmer record found',
+        farmer
+      });
+    }
```

Comment:
Can you explain what your `if` block is doing
You checking if `id` exists instead if farmer exists